code/modules/modular_computers/file_system/programs/robocontrol.dm

```diff
@@ -35,20 +35,34 @@
 	botcount = 0
 	current_user = user
 
-	for(var/B in GLOB.bots_list)
-		var/mob/living/simple_animal/bot/Bot = B
-		if(!(Bot.bot_mode_flags & BOT_MODE_ON) || Bot.z != zlevel || !(Bot.bot_mode_flags & BOT_MODE_REMOTE_ENABLED)) //Only non-emagged bots on the same Z-level are detected!
+	for(var/mob/living/simple_animal/bot/Bot as anything in GLOB.bots_list)
```

Comment:
```suggestion
	for(var/mob/living/simple_animal/bot/bot as anything in GLOB.bots_list)
```

Reply:
also tweak the variable name in the for proc as well

Reply:
all done